### ISSUE TITLE
[BUG] Dao parameter used mutliple times causes generator error

### ISSUE THREAD

**Issue by jerzykiler** [vitusortner/floor#179]
In code:
```
  @Query("""
    SELECT * FROM 
    mytable s 
    LEFT JOIN my_other_table ss ON ss.id = s.other_id
      AND s.spec_id = :spec_id
      AND ss.spec_id = :spec_id
  """)
  Future<List<MyEntity>> getMyEntitiesBySpecId(int spec_id);
```
I used `:spec_id` twice and it caused:
```
[SEVERE] floor_generator:floor_generator on lib/models/floor_database/database.dart:
Error running FloorGenerator
SQL query arguments and method parameters have to match.
package:my_flutter/models/dao/my_entity_dao.dart:28:31
   ╷
28 │   Future<List<MyEntity>> getMyEntitiesBySpecId(
   │                               ^^^^^^^^^^^^^^^^^^^^^^^
   ╵
```
If I use it only once then there is no error.

**Comment by vitusortner** [comment]
Right now, parameters can only be used once in a query. Thus, you would need to pass the same value to the function multiple time. Related issue #142

**Comment by mqus** [comment]
Closing as a duplicate of #142
